app/src/main/java/app/revanced/manager/ui/component/AppScaffold.kt

```diff
@@ -69,7 +69,11 @@ fun AppTopBar(
         scrollBehavior = scrollBehavior,
         navigationIcon = {
             if (onBackClick != null) {
-                IconButton(onClick = onBackClick) {
+                TooltipIconButton(
+                    modifier = Modifier,
```

Comment:
Explicitly passing empty `Modifier`s is unnecessary since the parameter should be set by default. Please remove these throughout the codebase.